internal/engine/upper_test.go

```diff
@@ -1620,6 +1631,20 @@ func TestNewMountsAreWatched(t *testing.T) {
 	})
 }
 
+func TestDockerComposeRecordsLogs(t *testing.T) {
+	f := newTestFixture(t)
+	m := f.setupHAProxy()
+	expected := "spoonerisms_1  | 2018-12-20T16:11:04.070480042Z yarn install v1.10."
```

Comment:
...spoonerisms? 🤔

Reply:
also wait i'm confused, is this actually running `docker-compose up`??!

Reply:
nvm i see! if you wanna be extra clear, make this bit its own paragraph with comment (like "call to `docker-compose logs` will return this line -- make sure log watcher picks it up")

also -- test that it ends up in manifest...logs? (Or todo for that test.)